builds/misc/templates/build-packages.yaml

```diff
@@ -310,13 +310,14 @@ stages:
           # build iot-identity-service here since they cannot build native aarch64 packages
           - bash: |
               set -ex
+              packageVersion=$(git tag | grep 1.2.[0-9]*$ | tail -1)
               git clone --recurse-submodules https://github.com/Azure/iot-identity-service.git -b release/1.2
```

Comment:
Is this git tag from /iotedge or /iot-identity-service repository?

Reply:
ah that one would be for iotedge. just moved this to be in the iot-identity-service directory to retrieve the correct tabs